FueledUtils/CollectionExtensions.swift

```diff
@@ -0,0 +1,9 @@
+public extension Collection {
+	public func get(_ index: Self.Index) -> Self.Iterator.Element? {
```

Comment:
Shall we add a documentation comment or rename it to something like getSafely(at index:)? (I think it'd be more like a standard library method with a label)

Reply:
Makes sense to me, making the change